definitions/modules/navigation.yaml

```diff
@@ -6,20 +6,82 @@ shared:
     var_type: f64
     event_name: KOHLSMAN_SET
     event_param: 1
-    multiply_by: 16
+  -
+    type: NumSet
+    var_name: A:KOHLSMAN SETTING MB:2
+    var_units: Millibars
+    var_type: f64
+    event_name: KOHLSMAN_SET
+    event_param: 2
+  -
+    type: event
+    event_name: BAROMETRIC
   -
     type: var
     var_name: A:AIRSPEED TRUE CALIBRATE
     var_units: Degrees
+    var_type: f64
+  -
+    type: var
+    var_name: A:ATTITUDE BARS POSITION
+    var_units: Percent
+    var_type: f64
+  -
+    type: NumIncrement
+    var_name: A:NAV OBS:1
+    var_units: Degrees
     var_type: i32
+    up_event_name: VOR1_OBI_INC
+    down_event_name: VOR1_OBI_DEC
+    increment_by: 1
+  -
+    type: NumIncrement
+    var_name: A:NAV OBS:2
+    var_units: Degrees
+    var_type: i32
+    up_event_name: VOR2_OBI_INC
+    down_event_name: VOR2_OBI_DEC
+    increment_by: 1
   -
     type: NumIncrement
+    var_name: ADF CARD
+    var_units: Degrees
+    var_type: i32
+    up_event_name: ADF_CARD_INC
+    down_event_name: ADF_CARD_DEC
+    increment_by: 1
+
+# - # Recommended not to use as this updates constantly, calibrating the heading indicator isn't used often
+#   type: var
+#   var_name: A:PLANE HEADING DEGREES GYRO
+#   var_units: Degrees
+#   var_type: i32
+
+init:
```

Comment:
why is there an init needed here? init should only be reserved for definitions that should only sync **on initial connection *ONLY***. you should get rid of everything below here. having definitions under shared will sync upon initial connection anyway.